pennylane/qnode.py

```diff
@@ -283,14 +292,23 @@ def construct(self, args, **kwargs):
             temp = [Variable(idx, name=key) for idx, _ in enumerate(_flatten(val))]
             kwarg_variables[key] = unflatten(temp, val)
 
+        Variable.free_param_values = np.array(list(_flatten(args)))
```

Comment:
does `_flatten` not return a list or array??

Reply:
Unfortunately `_flatten` returns a generator (which makes sense for most cases). Since NumPy v1.10 or so, you can no longer instantiate a NumPy array from a generator, hence the intermediate list conversion.